src/synthesizer/load_data/utils.py

```diff
@@ -28,13 +28,19 @@ def get_len(Length):
     return begin, end
 
 
-def age_lookup_table(cosmo, delta_a=1e-3, low_lim=1e-4):
+def age_lookup_table(cosmo, redshift=0., delta_a=1e-3, low_lim=1e-4):
```

Comment:
Should redshift not be a positional argument? There will always be an associated snapshot with a redshift.

Reply:
I agonised over this a bit. Settled on this as I wanted to make it as general as possible. The sim may be a lightcone, so you may just want to produce a lookup grid at all times. You may also wish to produce a single lookup grid for a sim for all snapshots, to further reduce compute. With this you can do this, without having to explain a default

Reply:
All good points! Fair play.